webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeSegmentStatisticsService.scala

```diff
@@ -109,7 +109,7 @@ class VolumeSegmentStatisticsService @Inject()(volumeTracingService: VolumeTraci
       )
     }.toList
     for {
-      (data, _) <- if (tracing.mappingIsEditable.getOrElse(false))
+      (data, _) <- if (tracing.hasEditableMapping.getOrElse(false))
```

Comment:
`getHasEditableMapping` can be used here as well (afaik)

```suggestion
      (data, _) <- if (tracing.getHasEditableMapping)
```